src/components/NpsSurvey.js

```diff
@@ -0,0 +1,147 @@
+import _ from 'lodash/fp'
+import { Component } from 'react'
+import { div, h, input } from 'react-hyperscript-helpers'
+import Interactive from 'react-interactive'
+import { buttonSecondary, Clickable } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { TextArea } from 'src/components/input'
+import { Ajax } from 'src/libs/ajax'
+import { authStore } from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+
+
+export const responseRequested = Utils.atom(false)
+
+const style = {
+  questionLabel: { fontWeight: 600, marginBottom: '0.5rem' },
+  questionInput: { marginBottom: '0.75rem', height: '4rem' }
+}
+
+export const NpsSurvey = _.flow(
+  Utils.connectAtom(responseRequested, 'responseRequested'),
+  Utils.connectAtom(authStore, 'authState')
+)(class NpsSurvey extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { requestable: false, expanded: false }
+  }
+
+  componentDidMount() {
+    const { authState: { isSignedIn } } = this.props
+
+    if (isSignedIn) {
+      this.loadStatus()
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    const { authState: { isSignedIn } } = this.props
+
+    if (isSignedIn && !prevProps.authState.isSignedIn) {
```

Comment:
This should be looking at `registrationStatus`, since that's the signal for the user being fully signed in and enabled.